sass/components/covid/dashboard.scss

```diff
@@ -3,4 +3,10 @@
     padding: 0 15px;
     width: 13%;
     float: left;
-  }
\ No newline at end of file
+  }
+
+div[data-qa='refusingToShield'] .govuk-warning-text__icon {
```

Comment:
Can see your predicament but it's best not to use anything other than styling classes for applying styles to an element.  If someone decides to change that `data-qa` tag in future, they're unlikely to know that it will have any effect on how something looks.

I'll help you have a look into what the actual problem is.

Reply:
agreed